ext/recaptcha/CRM/Utils/ReCAPTCHA.php

```diff
@@ -62,8 +62,7 @@ public static function hasSettingsAvailable() {
    * Check if reCaptcha has to be added on form forcefully.
    */
   public static function hasToAddForcefully() {
-    $config = CRM_Core_Config::singleton();
-    if (!$config->forceRecaptcha) {
+    if (!(bool)\Civi::settings()->get('forceRecaptcha')) {
```

Comment:
It doesn't like the spacing, but you could go one further and replace the whole if block with `return (bool) \Civi::settings()->get('forceRecaptcha');`

Reply:
Agreed, thanks @demeritcowboy